pkg/cert/util.go

```diff
@@ -202,7 +204,7 @@ func MapToX509CertChain(jwkAsMap map[string]interface{}) ([]*x509.Certificate, e
 	if err != nil {
 		return nil, err
 	}
-	return GetX509ChainFromHeaders(key)
+	return key.X509CertChain(), nil
```

Comment:
This replaces the use of the `GetX509ChainFromHeaders` func. Is the `GetX509ChainFromHeaders` func still used anywhere else, it would be nice if it could be removed!

Reply:
Yes, `GetX509ChainFromHeaders` is stil used in the JWS/JWK context. This here is a JWK.